.github/setup.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# This should be Fstar/.github/
+cwd=$(cd $(dirname $0); pwd -P)
```

Comment:
why not just `$(dirname $0)` or `$(realpath $(dirname $0))`?

Reply:
I remember problems with the other versions on some systems... can't recall what exactly though... Since the variety of CI machines here is pretty limited I guess both of them should be fine as well :)

Reply:
meh no need to change if the current system works -- was just curious